.github/workflows/turing.yaml

```diff
@@ -12,9 +12,11 @@ on:
       - "docs/**"
       - "engines/batch-ensembler/**"
       - "sdk/**"
+      - "infra/cluster-init/**"
```

Comment:
Since `cluster-init/**` is involved in this workflow (you are building its docker image for the e2e test), I think that it should not be excluded here.

Reply:
Also, it's expected that `path-ignore` at L11 is the same as `path-ignore` at L25 (same criteria for `push` and `pull_request`)

Reply:
Oh I see. Removing `cluster-init/**` from `paths-ignore`